PrismSampleApp/PrismSampleApp/ViewModels/LoginPageViewModel.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using Xamarin.Forms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using System.Threading.Tasks;
+using Prism.Services;
+using PrismSampleApp.Views;
+using PrismSampleApp.Services.Interfaces;
+using PrismSampleApp.Repository.Interfaces;
+using PrismSampleApp.Model;
+using PrismSampleApp.Resx;
+using Prism.AppModel;
+using Xamarin.Essentials;
+
+namespace PrismSampleApp.ViewModels
+{
+    public class LoginPageViewModel :ViewModelBase, IPageLifecycleAware
```

Comment:
Inherit IPageLifecycleAware, INavigatedAware in ViewMoedl base and override OnNavigated methods wherever you needed

Reply:
Inherited IPageLifecycleAware, INavigatedAware in ViewModelBase and overrided the code